opensrp-anc/src/main/java/org/smartregister/anc/library/AncLibrary.java

```diff
@@ -113,6 +124,12 @@ public static void init(@NonNull Context context, int dbVersion, @NonNull Activi
         }
     }
 
+    public static void init(@NonNull Context context, int dbVersion, @NonNull ActivityConfiguration activityConfiguration, @Nullable SubscriberInfoIndex subscriberInfoIndex, @Nullable RegisterQueryProvider registerQueryProvider, @Nullable AncMetadata metadata) {
```

Comment:
Please add java docs.

Reply:
Please update the java doc to explain what implementation is possible & a description of the parameters